### PR TITLE
Reorder head elements in base.html

### PR DESCRIPTION
**Pull request by szepeviktor**
There is a problem: in `theme-detection.js` things are wrapped in an IIFE, so it won't execute immediately.

### CODE REVIEW

**Comment by szepeviktor** [comment]
... I am not sure. Maybe the opposite is true.

**Comment by szepeviktor** [comment]
> things are wrapped in an IIFE, so it won't execute immediately.

False!
Just tested with IIFE in a file, and a simple function call inline after that.

File: src/_includes/layout/base.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-        {% include "partial/preload.html" %}
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="alternate" type="application/rss+xml" href="{{ site.url }}/feed.xml" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
         {% include "partial/meta.html" %}
+        <link rel="alternate" type="application/rss+xml" href="{{ site.url }}/feed.xml">
+
+        <script src="/js/theme-detection.js"></script>
```
Comment (szepeviktor) [inline src/_includes/layout/base.html:10]: Meta elements above this cause no delay in execution.

**Comment by adamlaki** [comment]
Thanks Viktor, I will add these fixes to our 11ty-starter.

**Pull request merged**